tests/TestCase/ORM/EntityTest.php

```diff
@@ -1060,6 +1103,29 @@ public function testToArrayVirtualProperties()
     }
 
     /**
+     * Tests setting virtual properties with merging.
+     *
+     * @return void
+     */
+    public function testSetVirtualWithMerge()
```

Comment:
Those already exist (with their former name without set prefix).
No one added the ones for the new feature however, and it was already broken by design.